docs/setup.md

```diff
@@ -0,0 +1,88 @@
+# Local Development Environment
+
+## Requirements
+
+### Terminus
+
+Pantheon's Terminus is an almost-essential command-line tool for managing sites. The CLI is required to run many of the scripts for building, updating, and deploying changes between the local development environment and the Pantheon platform.
+
+1. [Install Terminus](https://pantheon.io/docs/terminus/install/)
+2. [Setup a valid machine token](https://pantheon.io/docs/machine-tokens)
+3. Log in to terminus using the machine token\
+```terminus auth:login --email=<email@example.com> --machine-token=<machine_token>```
+4. [Review documentation](https://pantheon.io/docs/terminus/) to get started with the CLI
+
+### Docker
+
+Docker is an open-source container system for setting up virtualized environments. Containers are hardware-agnostic and platform-agnostic meaning they can run anywhere.
+
+1. [Install Docker](https://docs.docker.com/install)
+2. Increase your resources: Locate the 'Resources' section in your Docker preferences. For most architectures, this project requires at least 3GB of memory and 3 CPUs. Additional CPUs and memory may be helpful but should stay under the halfway mark of your total available resources.
+3. Disable the _'Start Docker when you log in'_ setting under the 'General' tab.
+
+### Lando
+
+This project supports development with Lando using the Pantheon recipe. This produces a local development environment that matches the hardware and configuration of the Pantheon platform. Most local tooling, including Composer, Drush, Drupal Console, and node applications are available through this recipe.
+
+1. [Review the preflight checks](https://docs.devwithlando.io/installation/preflight.html)
+2. [Download and install the latest release](https://github.com/lando/lando/releases)
+3. [Optional: Setup local certificate authority](https://docs.devwithlando.io/config/security.html)
```

Comment:
Should local CA setup be mandatory? I think you will get cert errors without this being a mandatory step since all references below this use https:// URLs.

Reply:
I will remove the 'optional' phrase. This is best to have enabled. It's something that I have a hard time setting up personally, but it is best practice.